packages/f36-ai-components/stories/AIChatInput.stories.tsx

```diff
@@ -0,0 +1,174 @@
+import type { Story } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+
+import {
+  AIChatInput,
+  type AIChatInputProps,
+} from '../src/AIChatInput/AIChatInput';
+import { Select } from '@contentful/f36-forms';
+import { PlusIcon } from '@contentful/f36-icons';
+import { IconButton } from '@contentful/f36-button';
+import { Tooltip } from '@contentful/f36-tooltip';
+import { Flex } from '@contentful/f36-core';
+import { range } from 'lodash';
+import Mention from '@tiptap/extension-mention';
+import { List } from '@contentful/f36-list';
+import { Editor } from '@tiptap/react';
+
+const DEFAULT_ARGS: Partial<AIChatInputProps> = {
+  style: { width: '400px' },
+  placeholder: 'Type your message...',
+};
+
+const Template: React.FC<
+  Omit<AIChatInputProps, 'setValue' | 'isSteaming' | 'onSubmit' | 'onStop'>
+> = (args) => {
+  const [isSteaming, setIsSteaming] = React.useState(false);
```

Comment:
I can't seem to toggle the `isStreaming` control in Storybook. I faced a similar issue with my PR and have [a solution](https://github.com/contentful/forma-36/blob/520b42e9d8c0e82d1b05dbf83725d5c69aaf540b/packages/f36-ai-components/stories/AIChatLayout.stories.tsx#L56-L62) using `useEffect`.